.github/workflows/update-helm-index.yml

```
@@ -46,7 +46,14 @@ jobs:
           body: |
             Hi.
 
-            We released a new version of Inspektor Gadget, so the helm repository index needs to be updated.
+            We released a new version of Inspektor Gadget [${{ steps.bump_inspektor_gadget.outputs.version }}](https://github.com/inspektor-gadget/inspektor-gadget/releases/tag/${{ steps.bump_inspektor_gadget.outputs.version }}), so helm index needs to be updated.
+
+            This PR was automatically generated by a CI cron job:
+            * [update-helm-index.yml](https://github.com/inspektor-gadget/charts/blob/main/.github/workflows/update-helm-index.yml)
+
+            This PR will not be automatically merged. Please review, edit and merge as appropriate.
+            After merging, the following will be updated:
+            * [GitHub Page for Charts](https://github.com/inspektor-gadget/charts/actions/workflows/pages/pages-build-deployment)
```

Did you mean to use this link instead?
```suggestion
            * [GitHub Page for Charts](https://github.com/inspektor-gadget/charts)
```

---

Right. Done, thanks!